docs/topic-guides/blendable-web-apps/localization.md

```diff
@@ -0,0 +1,115 @@
+# Localizing Starcounter Apps
+
+Real world applications often need to support a variety of languages. This article describes how to add such support to your application. It doesn't describe any practices for translation of texts or internals of how localization support works in Starcounter.
+
+## Getting started
+
+1. Create resources file with strings you want to localize.
+   1. In Visual Studio, open "Add new item" menu and choose "Resources File"
+   1. Select any name for your file. We'll use `Strings.resx` in this example
```

Comment:
These two bullet points don't end with a period. That's inconsistent with the other bullet points.